### ISSUE TITLE
[BUG] False positive for non-equal string comparison

### ISSUE THREAD

**Issue by nunnatsa** [nunnatsa/ginkgolinter#82]
ginkgolinter wrongly trigger an error for string non-equal comparison; e.g. for
```golang
Expect(str1 < str2).Should(BeTrue())
```

The linter trigger warning and suggests:
```golang
Expect(str).Should(BeNumerically("<", str2))
```

**Environment:**
 - OS: linux
 - Version: v0.11.1
